MyBlog.Services/Areas/Articles/ArticleService.cs

```diff
@@ -125,21 +121,21 @@ public async Task DeleteByIdAsync(int id, JWTInfo userJwtInfo)
         await _context.SaveChangesAsync();
     }
 
-    private async Task<bool> CheckTitleAreBusyAsync(int id, ArticleDtoInput articleInput)
+    private async Task<bool> CheckArticleTitleAreFreeAsync(int id, ArticleDtoInput articleInput)
     {
-        var article = await GetArticleByIdAsync(id);
+        var articleById = await GetArticleByIdAsync(id);
```

Comment:
rename this method - you return bool type
the name of your method should give me the answer what is going on

Reply:
also, think about articleById variable renaming